docs/content/faq.md

```diff
@@ -71,7 +71,9 @@ proper authentication.
 
 ### Is there support for monitoring and observability for Proxy?
 
-Right now, we have structured logs for proxy. Along with that, we have added tracing to help developers identify critical code paths and debug latency issues. While there is no setup required for logs, tracing requires some installation. We currently store our trace exports using [Jaeger](https://www.jaegertracing.io/). Further support for other exporters is in progress.  To try out tracing, do the following:
+Right now, we have structured logs for proxy. Along with that, we have added tracing to help developers identify critical code paths and debug latency issues. While there is no setup required for logs, tracing requires some installation. We currently store our trace exports using [Jaeger](https://www.jaegertracing.io/), GCP Stackdriver & Datadog (untested). Further support for other exporters is in progress.  
```

Comment:
What do you think about changing `We currently store our trace exports using [Jaeger](https://www.jaegertracing.io/), GCP Stackdriver & Datadog (untested).` to `We currently store our trace exports using [Jaeger](https://www.jaegertracing.io/), with GCP Stackdriver & Datadog (untested) as optional alternatives.`

Reply:
Not really, we have not updated our helm charts to include Jaeger, so technically all 3 of them are optional. 

I have created an issue for this at https://github.com/gomods/athens/issues/762

Reply:
Ok, maybe then the line should say something like `We currently support exporting tracing with Jaeger, GCP Stackdriver and Datadog (untested).` Still with links is awesome though.

Reply:
I added the links and updated the language as you told. Thanks for reviewing it!